frontend/src/components/Permissions.tsx

```diff
@@ -1,26 +1,19 @@
 import { PlusCircle } from "lucide-react";
 import React, { useEffect, useState } from "react";
+import BudgetAmountSelect from "src/components/BudgetAmountSelect";
+import BudgetRenewalSelect from "src/components/BudgetRenewalSelect";
```

Comment:
Can we leave the refactoring here for now? This might bring conflicts in https://github.com/getAlby/hub/pull/14 🥲

Reply:
I don't think that makes sense, no. It's literally just encapsulating markup and, no changes to state variables and such.